Applications/Spire/Source/Ui/Tooltip.cpp

```diff
@@ -10,7 +10,7 @@ namespace {
     auto style = StyleSheet();
     style.get(Any()).
       set(BackgroundColor(QColor::fromRgb(0, 0, 0))).
-      set(border_size(0));
+      set(border_color(QColor::fromRgb(0, 0, 0)));
```

Comment:
What part of the spec does this correspond to? I don't see any border color styling and the border size is supposed to be 0.